tool/matomo/templates/tracking_code.mustache

```diff
@@ -51,10 +51,17 @@
 
     function embedTrackingCode() {
         var u='//{{{siteurl}}}/';
+        var p='//{{{piwikurl}}}/';
+        var piwik = u;
+
+        if (p != '///') {
```

Comment:
Id simplify this and do that on the php side, see

https://github.com/catalyst/moodle-tool_webanalytics/pull/19/files#r311871053